espGarden/main/include/heltec_unofficial.h

```diff
@@ -292,6 +292,53 @@ void display_centered_string(const char *str, uint8_t font_size, uint32_t durati
 
 }
 
+void display_stats() {
+    char ip[16] = {0};
+    wifi_manager_get_ip(ip);
+
+    int32_t rssi = wifi_manager_get_rssi();
```

Comment:
if we want could just make rssi a pointer or the other way around for ip. kinda bugs me that get ip and get rssi don't work in the same way. not exactly sure why I did it this way, I remember something about made printing it on screen easier.

Reply:
Could be a nice refactor if you'd like